ledger/heavy/proc/replication.go

```diff
@@ -136,7 +148,7 @@ func storeDrop(
 	}
 	err = drops.Set(ctx, *d)
 	if err != nil {
-		return nil, errors.Wrapf(err, "heavyserver: drop storing failed")
```

Comment:
Why do you remove wrapping?

Reply:
it was useless, we already adding context in `store` method